types/node-forge/index.d.ts

```diff
@@ -17,6 +17,7 @@
 //                 Ligia Frangello     <https://github.com/frangello>
 //                 Dmitry Avezov       <https://github.com/avezov>
 //                 Jose Fuentes        <https://github.com/j-fuentes>
+//                 Anya Reyes          <https://github.com/darkade>
```

Comment:
Should the version in the header (i.e. above) be updated to match the readme?

Reply:
Ohh yeah, I think you are right. I'm going to update that